javascript-light-bulb/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./bulb.css" />
+    <title>Light Bulb Switcher</title>
+  </head>
+  <body>
+    <div id="flexing-bulb">
+      <div class="Main_Bulb"></div>
+    </div>
+
+    <script>
+      const mainBulb = document.querySelector('.Main_Bulb');
+
+      mainBulb.className === 'Main_Bulb';
+
+      mainBulb.addEventListener('click', () => {
+        if (mainBulb.className === 'Main_Bulb') {
+          document.body.style.backgroundColor = '#000';
+          mainBulb.className = 'Main_Bulb grey';
+          mainBulb.style.boxShadow = '2px 2px 1rem #f4f4f4';
+        } else if (mainBulb.className === 'Main_Bulb grey') {
+          document.body.style.backgroundColor = '#fff';
+
+          mainBulb.className = 'Main_Bulb';
+          mainBulb.style.boxShadow = '6px 12px 1rem black';
+        }
```

Comment:
No inline styling. You want to avoid inline styling. You should be adding or removing classes

Reply:
![Recording 2023-11-07 at 11 19 38 (1)](https://github.com/Hagop420/c1023-code-solutions/assets/81959688/9e80b7d8-5ec2-4fa3-8dc5-702ac0341531)